wallet/src/actors/worker/methods.rs

```diff
@@ -1153,3 +1194,10 @@ impl Worker {
         wallet.export_master_key(password).map_err(Error::from)
     }
 }
+
+fn is_old_epoch(birth_date: u32, checkpoint_zero_timestamp: i64, checkpoints_period: u16) -> bool {
+    let seconds_from_genesis = chrono::offset::Utc::now().timestamp() - checkpoint_zero_timestamp;
```

Comment:
We use our custom `get_timestamp` function

```suggestion
    let seconds_from_genesis = get_timestamp() - checkpoint_zero_timestamp;
```